packages/terra-clinical-item-display/src/ItemDisplay.jsx

```diff
@@ -43,6 +44,16 @@ const propTypes = {
    * One of `'center'`, `'top'`, `'inline'`.
    */
   iconAlignment: PropTypes.oneOf(['center', 'top', 'inline']),
+  /**
+   * The meaning of the text based on styling, used by screen readers.
+   * Changing `meaning` will not visually change the style of the content.
+   */
```

Comment:
Unless it was omitted in purpose, it might be worth adding a line here, something like: `For strike through text style only, the default value is 'deletion'`. Currently the Display Props Table says that `meaning` has no default value (which is true for all cases, but strike through text).

Reply:
I can add a note about strikethrough to the prop description.